vizro-ai/hatch.toml

```diff
@@ -29,12 +29,9 @@ dependencies = [
   "langchain-anthropic",
   "pre-commit"
 ]
+env-vars = {VIZRO_AI_LOG_LEVEL = "DEBUG"}
 installer = "uv"
 
-[envs.default.env-vars]
-UV_PRERELEASE = "allow"
-VIZRO_AI_LOG_LEVEL = "DEBUG"
-
```

Comment:
I've seen similar changes about UV_PRERELEASE a few times in different PRs. Can you just quickly explain what these changes are supposed to do? 😄

Reply:
So basically `UV_PRERELEASE = "allow"` is like setting `--pre` on `pip install` and means that you get pre-releases of packages, e.g. pydantic sometimes do alpha release that by default you don't get without explicitly allowing them.

In principle it's a good idea to use prereleases in CI because then if a forthcoming release of one of your dependencies breaks your tests you have more notice and can try and fix it in advance of the actual release coming out and affecting users. In reality though I found that a plotly alpha release broke all our tests in a way that will not be the case in their stable release. So testing against prereleases wasn't possible in the end.